src/components/People.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { PersonLink } from './PersonLink';
+import { getPeopleWithParents } from '../utils/utils';
+
+export const People: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[] | null>(
+    null,
+  );
+  const [error, setError] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const isArr = Array.isArray(peopleFromServer);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const data = await getPeople();
+
+        if (Array.isArray(data)) {
+          setPeopleFromServer(getPeopleWithParents(data));
+          setIsLoading(true);
+        }
+      } catch {
+        setError('ERROR');
+        setIsLoading(true);
+      }
+    };
+
+    fetchPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {!isLoading && <Loader />}
+
+          {error === 'ERROR' && (
+            <>
+              <p data-cy="peopleLoadingError" className="has-text-danger">
+                Something went wrong
+              </p>
+
+              <p data-cy="noPeopleMessage">There are no people on the server</p>
+            </>
+          )}
```

Comment:
what do you mean here, error can be something different than `'ERROR'`? looks like it can be boolean state and if `isError` === true then you can show the error message